autoscheduler/frontend/src/App.tsx

```diff
@@ -1,13 +1,24 @@
 import * as React from 'react';
 import { Router, Link } from '@reach/router';
+import { AppBar, IconButton, Toolbar} from '@material-ui/core'
+/*import AppBar from '@material-ui/core/AppBar';
+import IconButton from '@material-ui/core/IconButton';
+import Toolbar from '@material-ui/core/Toolbar';
+*/
```

Comment:
Yes, it is: importing multiple exports adds ~200kB when compiled with `npm run build` over importing them separately, and I meant to make a comment about it: I think we should use babel to get around this, since it's a pretty significant difference.

Reply:
Yeah that is pretty significant, should we make this a task? 

Also have you used babel before @firejake308 ? I thought it just converts ES6 syntax to ES5 for old browsers. Can webpack not accomplish this?

Reply:
Yeah, I'm in favor of using Babel to reduce our bundle size